domains/zayaanar.json

```diff
@@ -7,7 +7,9 @@
         },
     
         "record": {
-            "CNAME": "userandaname.github.io"
+            "CNAME": ["userandaname.github.io"],
+            "MX": ["mx.zoho.com", "mx2.zoho.com", "mx3.zoho.com"],
+            "TXT": ["v=spf1 include:zohomail.com ~all"]
```

Comment:
You cannot use CNAME with MX and TXT records, also CNAME isn't supposed to be in a array